spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/DefaultMessagePropertiesConverter.java

```diff
@@ -91,17 +91,30 @@ public DefaultMessagePropertiesConverter(int longStringLimit, boolean convertLon
 	}
 
 	/**
+	 * @param correlationIPolicy true to use.
+	 * @see #setCorrelationIdPolicy(CorrelationIdPolicy)
+	 * @deprecated - the byte[] version of correlation id will be removed in 2.0
+	 */
+	@Deprecated
+	public void setCorrelationIdAsString(CorrelationIdPolicy correlationIPolicy) {
+		this.correlationIdPolicy = correlationIPolicy;
+	}
+
+	/**
 	 * For inbound, determine whether correlationId, correlationIdString or
 	 * both are populated. For outbound, determine whether correlationIdString
 	 * or correlationId is used when mapping; if {@code CorrelationIdPolicy.BOTH}
 	 * is set for outbound, String takes priority and we fallback to bytes.
 	 * Default {@code CorrelationIdPolicy.BYTES}.
 	 * @param correlationIPolicy true to use.
+	 * @deprecated - the byte[] version of correlation id will be removed in 2.0
 	 */
-	public void setCorrelationIdAsString(CorrelationIdPolicy correlationIPolicy) {
-		this.correlationIdPolicy = correlationIPolicy;
+	@Deprecated
+	public void setCorrelationIdPolicy(CorrelationIdPolicy correlationIPolicy) {
```

Comment:
OK. Let it be. Makes sense, although doesn't provide much value if it deprecated anyway.
I see you remove `CorrelationIdPolicy`, so I mark it with `@Deprecated` and merge.
Anything else to pay attention?